packages/node/test/unit/dev.test.ts

```diff
@@ -117,7 +117,8 @@ async function withDevServer(
         { runningTimeout: 300 }
       ));
 
-    test('with `waitUntil` from context rejecting a promise ', () =>
```

Comment:
not blocking, but I think I'd generally prefer this workaround to be its own PR. But we can discuss this process on Monday!